### PR TITLE
[PRM-690] - Endpoint to expose API

### PR DESCRIPTION
**Pull request by LewisMitchell**

### CODE REVIEW

File: app/utils/RegimeHelper.scala [hunk 1]
```diff
@@ -47,4 +47,6 @@ object RegimeHelper {
       case _ => ""
     }
   }
+
+  def constructMTDVATEnrolmentKey(vrn: String): String = s"HMRC-MTD-VAT~VRN~$vrn"
```
Comment (ruthdas03) [inline app/utils/RegimeHelper.scala:51]: As in the  schema the  pattern for VRN is "^([a-zA-Z0-9]{1,20})$", do this need to be validated against this pattern

**Review by ruthdas03 — COMMENTED** [review]
Just one observation. Rest is ok.

File: app/utils/RegimeHelper.scala (hunk 1 above)
Comment (ruthdas03) [reply app/utils/RegimeHelper.scala:51]: Do we not need a test case for this method.

Comment (LewisMitchell) [reply app/utils/RegimeHelper.scala:51]: Checking the need for validation now...

Will add tests cases - I didn't add them before due to the lack of tests for the helper, but worth adding them

Comment (LewisMitchell) [reply app/utils/RegimeHelper.scala:51]: Pushed some tests, waiting on an answer for validation

Comment (LewisMitchell) [reply app/utils/RegimeHelper.scala:51]: Validation should be performed - I will validate it against: https://github.com/hmrc/service-enrolment-config/blob/d95a95f054c1638049677f69c7952b8ff00ac2e7/conf/SEC1_with_enrolment_rules_json/prod/HMRC-MTD-VAT.json#L7

**Pull request merged**